src/utils/IdealGraphVisualizer/Util/src/main/java/com/sun/hotspot/igv/util/PropertiesConverter.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package com.sun.hotspot.igv.util;
+
+import com.sun.hotspot.igv.data.Properties;
+import com.sun.hotspot.igv.data.Property;
+
+/**
+ *
+ * @author ksakata
+ */
+public class PropertiesConverter {
```

Comment:
This class should escape at least the most common HTML characters. See for example the difference in the `dump_spec` property in the tooltip and in the Properties window:

![different-dump_spec](https://user-images.githubusercontent.com/8792647/182841654-e552502d-99a9-44a1-8e19-5686563f1b63.png)

Luckily, there is already a helper method for escaping HTML: [com.sun.hotspot.igv.util.StringUtils::escapeHTML](https://github.com/openjdk/jdk/blob/ce61eb6ff99eaaece463091b8481e27f84f80684/src/utils/IdealGraphVisualizer/Util/src/main/java/com/sun/hotspot/igv/util/StringUtils.java#L33).

Reply:
Thank you for pointing this out! I've used that method. Special characters display correctly.
<img width="360" alt="スクリーンショット 2022-08-08 15 45 45" src="https://user-images.githubusercontent.com/60008/183360106-446226b3-5ae0-441c-bd14-78000c95f85f.png">

Unfortunately, apostrophes in the tooltip text were shown in the form of character entity reference. 
<img width="548" alt="スクリーンショット 2022-08-08 14 43 22" src="https://user-images.githubusercontent.com/60008/183359617-f6ba6be3-e2f6-450b-aae2-a8716eac0d69.png">
So I've changed StringUtils class to use numeric entity reference for apostrophe instead of character entity reference. After that, we can see apostrophes properly.
<img width="461" alt="スクリーンショット 2022-08-08 14 53 02" src="https://user-images.githubusercontent.com/60008/183360045-55702560-705d-4deb-bb90-b68b9580cf1b.png">